core/src/main/java/feast/core/model/FeatureSet.java

```diff
@@ -125,6 +132,24 @@ public void setProject(Project project) {
     this.project = project;
   }
 
+  public String getReference() {
+    return String.format("%s/%s", getProjectName(), getName());
+  }
+
+  /** @return Pair &lt;ProjectName, FeatureSetName&gt; */
+  public static Pair<String, String> parseReference(String reference) {
+    String[] split = reference.split("/", 2);
+    if (split.length == 1) {
+      return Pair.of(Project.DEFAULT_NAME, split[0]);
+    } else {
```

Comment:
I dont think we need an `else` if we are returning, and we should probably raise an exception in the case of >2.